src/main.zig

```diff
@@ -2180,6 +2180,18 @@ fn buildOutputType(
                             fatal("unsupported -undefined option '{s}'", .{it.only_arg});
                         }
                     },
+                    .rtlib => {
+                        // Unlike Clang, we support `none` for explicitly omitting compiler-rt.
+                        if (mem.eql(u8, "none", it.only_arg)) {
+                            want_compiler_rt = false;
+                        } else if (mem.eql(u8, "compiler-rt", it.only_arg) or
+                            mem.eql(u8, "libgcc", it.only_arg)) {
+                            want_compiler_rt = true;
```

Comment:
```suggestion
                            mem.eql(u8, "libgcc", it.only_arg))
                        {
                            want_compiler_rt = true;
```

Reply:
Real talk: I like `zig fmt`. I can write `zig fmt`-compliant code 99% of the time. *Except for this rule.* It's so bizarre and unintuitive and aesthetically displeasing. ugh.

Reply:
a) I disagree, I find it both logical and visually important; the entry to a new block should include an indentation change, so if part of the `if` condition is indented, the brace needs to wrap onto a line to have one un-indented line. I find code without it much more difficult to visually parse.

b) Just use `zig fmt` on save :p